Changes

```diff
@@ -166,6 +166,10 @@ Next version (4.05.0):
   a short description in overviews.
   (Florian Angeletti)
 
+- PR#7478, GPR#1037: ocamldoc, do not use as a module preamble documentation
+  comments that occur after the first module element
+  (Florian Angeletti, report by Daniel Bünzli)
+
```

Comment:
It does not seem completely clear-cut to me since documentations affected by the change were already somehow broken. Nevertheless, erring on the side of caution and adding a warning about open statements is probably better.